features/number_item.feature

```diff
@@ -128,5 +128,12 @@ Feature:  number_item
     When I deploy the rules file
     Then It should log "Number Two is positive? true" within 5 seconds
 
+  Scenario: NumberItem can be converted to Quantity
+    Given code in a rules file
+      """
+      logger.info("NumberTwo to °C: #{NumberTwo|"°C"} equals quantity: #{NumberTwo|"°C" == Quantity.new('70°C')}")
```

Comment:
We already did this right?  did we just not test for it?

Reply:
Yes, NumberItem already had the `|` operator to convert into quantity. I'm merely adding the test for it.